Assets/Scripts/Environment/Office/OfficeDoor.cs

```diff
@@ -6,8 +6,8 @@ public class OfficeDoor : InteractableObject
     [SerializeField] private Vector3 doorOpenOffset;
 
     public override void OnTriggerEnter(Collider other)
-    {
-        SetPlayerIsColliding(other);
+    {
+        base.OnTriggerEnter(other);
```

Comment:
For example. this might break if we change the base class. Not got chance to test it yet.

Reply:
Door working perfectly now.